src/dc/sample/ObjectScript.cls

```diff
@@ -2,12 +2,43 @@ Class dc.sample.ObjectScript
 {
 
 ClassMethod Test() As %Status
+{
+    set sc = $$$OK
+    Try {
+        set sc = ..Run() ; Python
+        set sc = ..Run(0) ; ObjectScript
+    }
+    Catch ex {
+        set sc=ex.AsStatus()
+    }
+    return sc
+}
+
+ClassMethod Run(python As %Boolean = 1) As %Status
+{
+    return:python ..Python()
+    return:'python ..ObjectScript()
+}
+
+ClassMethod ObjectScript() As %Status
 {
     set a=42 
-    write "Hello World!",!
+    write "Hello World! from ",$CLASSMETHOD("dc.sample.ObjectScript","%ClassName",1)," in ObjectScript language",!
     write "This is InterSystems IRIS with version ",$zv,!
-    write "Current time is: "_$zdt($h,2)
+    write "Current time is: "_$zdt($now(),2,,6),!
     return a
 }
 
+ClassMethod Python() As %Status [ Language = python ]
+{
+import iris    
+from datetime import datetime
+a=42 
+print("Hello World! from",iris.cls('dc.sample.ObjectScript')._ClassName(1),"in Python language")
+print("This is InterSystems IRIS with version",iris.cls('%SYSTEM.Version').GetVersion())
```

Comment:
'dc.sample.ObjectScript' can be replaced by `__name__`

`iris.cls('%SYSTEM.Version').GetVersion()` = `iris.system.Version.GetVersion()`

Reply:
Thanks !!